host/xtest/CMakeLists.txt

```diff
@@ -17,14 +17,15 @@ if(OPENSSL_FOUND)
 endif()
 
 include(GNUInstallDirs)
+include(${OPTEE_TEST_ROOT_DIR}/scripts/common.mk)
```

Comment:
Including a makefile (`.mk`) from a CMake file ? Does this work?

Reply:
you are right - i think this is against CMAKE design model.
Shall i just use `set(PYTHON3 python3)`  ?

Reply:
Yes or rather whatever the syntax is for "set PYTHON3 to python3 if it is not set". My CMake knowledge is a bit limited ;-)